cli/validate.go

```diff
@@ -34,6 +35,17 @@ func validateRun(cmd *cobra.Command, args []string) {
 
 		for _, file := range files {
 			if file.IsDir() && !strings.HasPrefix(file.Name(), ".") {
+				// If the directory is empty, don't validate it
+				subfiles, err := ioutil.ReadDir(file.Name())
+				if err != nil {
+					newLogger.Critical(newCtx, "%#v", maskAny(err))
```

Comment:
I'd use `Error` here, instead of `Critical`. I think `Critical` should be reserved for "the nukes are launching" level stuff.